readme.txt

```diff
@@ -1,8 +1,8 @@
 === Comet Cache ===
 
-Stable tag: 161119
+Stable tag: 161207
 Requires at least: 4.2
-Tested up to: 4.7
+Tested up to: 4.7-alpha
```

Comment:
Good catch. Hmm. I'm not sure how this got changed back to `-alpha` then. I don't remember changing that myself, but I guess I must have.